src/control/modify.js

```diff
@@ -267,6 +271,7 @@ class ModifyControl extends Control {
      */
     this.modifyInteraction = new Modify({
       features: this.selectModify.getFeatures(),
+      style: this.modifyStyle || undefined,
```

Comment:
modifyStyle is already undefined by default
```suggestion
      style: this.modifyStyle,
```